docs/howto/manage-applications.md

```diff
@@ -543,42 +506,44 @@ This will issue a warning with a list of all the pieces to be removed and a requ
 
 All associated resources will also be removed, provided they are not hosting containers or another application's units.
 
-If persistent storage is in use by the application it will be detached and left in the model; however, if you wish to destroy that as well, you can use the `--destroy-storage` option.
-
-```{note}
-It it normal for application removal to take a while (you can inspect progress in the usual way with `juju status`). However, if it gets stuck in an error state, it will require manual intervention. In that case, please run `juju resolved --no-retry <unit>` for each one of the application's units (e.g., `juju resolved --no-retry kafka/0`).
-```
-
-```{ibnote}
-See more: {ref}`command-juju-remove-application`
-```
-
+If persistent storage is in use by the application, it will be detached and left in the model; however, if you wish to destroy that as well, you can use the `--destroy-storage` option.
 
-````{note} Troubleshooting:
+If the application has relations with another application, this relation will be terminated (which may adversely affect the other application).
 
-````{dropdown} One or more units are stuck in error state
+Note: It it normal for application removal to take a while (you can inspect progress in the usual way with `juju status`). However, if it gets stuck in an error state, it will require manual intervention. In that case, please run `juju resolved --no-retry <unit>` for each one of the application's units (e.g., `juju resolved --no-retry kafka/0`).
```

Comment:
```suggestion
Note: It is normal for application removal to take a while (you can inspect progress in the usual way with `juju status`). However, if it gets stuck in an error state, it will require manual intervention. In that case, please run `juju resolved --no-retry <unit>` for each one of the application's units (e.g., `juju resolved --no-retry kafka/0`).
```